hermes/crates/cardano-chain-follower/src/utils.rs

```diff
@@ -91,3 +93,89 @@ pub(crate) fn blake2b_256(value: &[u8]) -> anyhow::Result<[u8; 32]> {
     b.try_into()
         .map_err(|_| anyhow::anyhow!("Invalid length of blake2b_256, expected 32 got {}", b.len()))
 }
+
+#[allow(dead_code)]
```

Comment:
It is better place all macro definitions near the code itself and split them with the comments

Reply:
This might be used in multiple places within the metadata (maybe other CIP), so these hash functions are place in the utils file